src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -27,13 +27,11 @@ internal sealed class BuildUpToDateCheck : OnceInitializedOnceDisposed, IBuildUp
         private const string TelemetryEventName = "UpToDateCheck";
         private const string Link = "Link";
 
-        private static readonly HashSet<string> s_knownOutputGroups = new HashSet<string>
-        {
-            "Symbols",
-            "Built",
-            "Documentation",
-            "LocalizedResourceDlls"
-        };
+        private static ImmutableHashSet<string> KnownOutputGroups => ImmutableHashSet<string>.Empty
```

Comment:
"readonly", static fields begin with "s_" according to the repo's [coding conventions](https://github.com/dotnet/project-system/blob/master/docs/repo/coding-conventions.md). Also, ImmutableHashSet has a method called, ImmutableHashSet.Create that you can call here.

Reply:
In this case, they're properties which I thought were upper cased, no prefix? Yeah, I used this style since I used it lower down with situations where I wanted to union in sets.